### ISSUE TITLE
Starred Playlist Not Returning All Songs

### ISSUE THREAD

**Issue by tsquillario** [tsquillario/Jamstash#224]
getStarred returns ALL starred artists, albums, songs. 
http://www.subsonic.org/pages/api.jsp#getStarred

I bet we are limiting the songs by the "Auto Playlist Size". We might as well display them.  All the songs are in the response anyway.

**Comment by Hyzual** [comment]
You're right, we do get all starred artists, albums and songs in one call.
However, I'm concerned about displaying them all. On my own server I've got around 2000 starred songs and I'm not sure adding 2000 items to the DOM will go smoothly.
What do you think about it ?
